test/core/gprpp/mpscq_test.cc

```diff
@@ -44,16 +46,16 @@ static test_node* new_node(size_t i, size_t* ctr) {
   return n;
 }
 
-static void test_serial(void) {
+TEST(MpscqTest, Serial) {
   gpr_log(GPR_DEBUG, "test_serial");
   MultiProducerSingleConsumerQueue q;
   for (size_t i = 0; i < 10000000; i++) {
     q.Push(&new_node(i, nullptr)->node);
   }
   for (size_t i = 0; i < 10000000; i++) {
     test_node* n = reinterpret_cast<test_node*>(q.Pop());
-    GPR_ASSERT(n);
-    GPR_ASSERT(n->i == i);
+    ASSERT_TRUE(n);
```

Comment:
Comparison against nullptr should probably be done explicitly.

Reply:
Done.